### PR TITLE
Update mkdocs.yml, fix broken link

### PR DESCRIPTION
**Pull request by aeng**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/zanata/zanata-client/pull/67" target="_blank">ReviewNinja

File: docs/commands/push.md [hunk 1]
```diff
@@ -57,4 +57,4 @@ zanata-cli pull -s src -t trans --create-skeletons
 
 The option `--create-skeletons` is used to make sure files will be written even if there are no translations.
 
-For more information on the pull command, see [Document Download with Client](commands/pull).
+For more information on the pull command, see [Document Download with Client](/commands/pull).
```
Comment (carlosmunoz) [inline docs/commands/push.md]: Should this end with `.md` too?

File: docs/installation.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ sudo yum install zanata-client
 
 ## Installation with Ivy
 
-See [Ivy installation](ivy/installation) for more information.
+See [Ivy installation](/ivy/installation) for more information.
```
Comment (carlosmunoz) [inline docs/installation.md]: `.md` at the end ?

File: docs/ivy/installation.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 The Ivy distribution of the client is a small script that will download the client the first time it is run. This distribution requires Apache Ivy to run.
 
-1. See [Setup Ivy](ivy/setup) for Apache Ivy installation.
+1. See [Setup Ivy](/ivy/setup) for Apache Ivy installation.
```
Comment (carlosmunoz) [inline docs/ivy/installation.md]: another one :point_up:

File: docs/maven-plugin/commands.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ mvn zanata:push -Dzanata.srcDir="src/messages"
 
 This will look for source strings in "src/messages".
 
-More detail on the push command can be found at [Document Upload with Client](commands/push)
+More detail on the push command can be found at [Document Upload with Client](/commands/push)
```
Comment (carlosmunoz) [inline docs/maven-plugin/commands.md]: :point_up:

File: docs/maven-plugin/commands.md [hunk 5]
```diff
@@ -60,4 +60,4 @@ To download the source documents as well, specify pull type 'both' as shown here
 mvn zanata:pull -Dzanata.pullType="both"
 ```
 
-More detail on the pull command can be found at [Document Download with Client](commands/pull)
+More detail on the pull command can be found at [Document Download with Client](/commands/pull)
```
Comment (carlosmunoz) [inline docs/maven-plugin/commands.md]: :point_up:

File: docs/maven-plugin/configuration.md [hunk 6]
```diff
@@ -1,11 +1,11 @@
 To use the Maven Plugin, some Zanata configuration files and an installation of Apache Maven are needed. Some additional configuration will let you use a shorter command to run the Maven Plugin.
 
-For Apache Maven installation, see [Installing the Maven Plugin](maven-plugin/installation).
+For Apache Maven installation, see [Installing the Maven Plugin](/maven-plugin/installation).
```
Comment (carlosmunoz) [inline docs/maven-plugin/configuration.md]: :point_up:

**Comment by carlosmunoz** [comment]
In some places we end link urls with `.md`, in others we don't. Lets have some consistency there if it's all the same.

**Comment by aeng** [comment]
Removed all .md for wiki reference.

**Comment by carlosmunoz** [comment]
:+1:

**Pull request merged**